src/main/scala/de/upb/cs/swt/delphi/instanceregistry/Docker/DockerActor.scala

```diff
@@ -0,0 +1,76 @@
+package de.upb.cs.swt.delphi.instanceregistry.Docker
+
+import akka.actor.{Actor, ActorLogging, ActorSystem, Props}
+import akka.stream.ActorMaterializer
+import de.upb.cs.swt.delphi.instanceregistry.Docker.DockerActor._
+import de.upb.cs.swt.delphi.instanceregistry.Registry
+import de.upb.cs.swt.delphi.instanceregistry.io.swagger.client.model.InstanceEnums.ComponentType
+
+import scala.concurrent.duration._
+import scala.concurrent.{Await, ExecutionContext}
+
+class DockerActor(connection: DockerConnection) extends Actor with ActorLogging {
+
+  implicit val system: ActorSystem = Registry.system
+  implicit val materializer: ActorMaterializer = Registry.materializer
+  implicit val ec: ExecutionContext = system.dispatcher
+  val container = new ContainerCommands(connection)
+
+  case class DockerClient(connection: DockerConnection)
+
+
+  log.info(s"DockerActor started")
```

Comment:
Is it possible to do an initial connection test to see whether the specified docker host is reachable ? Something like getting the version of the docker service would be ideal. If the docker host is not reachable, we should terminate the application.

Reply:
As discussed, this is not needed anymore, as all docker commands implement proper error handling and we do not want the application to terminate when docker is not reachable.